### PR TITLE
Remove browser code

### PR DESCRIPTION
**Pull request by danlekko**
- Update Lekko SDK protos
- Remove browser code
- Cleanup init client input options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,27 @@
...
-const lekko = require('@lekko/node-server-sdk');
-const client = await lekko.initApiClient(
-{
-    hostname: "hostname",
-    apiKey: "lekko_apikey",
-    repositoryOwner: "repo_owner",
-    repositoryName: "repo_name"
-}
-)
+const client = await initSidecarClient({
```
Comment (DavidSGK) [inline README.md:7]: Looks like the example code for API and sidecar are switched?

File: src/index.ts [hunk 2]
```diff
@@ -3,24 +3,35 @@ import { ClientContext } from './context';
...
 
-async function initApiClient(options: ClientOptions): Promise<TransportClient> {
-  if (!options.apiKey) {
-    throw new Error("apiKey is required for API Client");
-  }
-  const transport = await new ClientTransportBuilder(options.hostname, TransportProtocol.HTTP, options.apiKey).build();
+async function initAPIClient(options: APIOptions): Promise<TransportClient> {
+  const transport = await new ClientTransportBuilder(
+    {
+      hostname: "https://prod.api.lekko.dev",
```
Comment (DavidSGK) [inline src/index.ts:15]: Should we have this point to prod as default but allow pointing to staging (or a different hostname) based on an env variable or something? Not sure if that might be valuable for internal testing.

File: example/index.js [hunk 3]
```diff
@@ -1,23 +1,17 @@
...
-const app = express();
-const port = 3000;
-
-app.listen(port, async () => {
-  console.log(`Example app listening on port ${port}`);
   
+async function getFeature() {
   // Initialize the client, replace "repo_owner", "repo_name" and "lekko_api_key"
-  const client = await lekko.initApiClient(
+  const client = await lekko.initAPIClient(
```
Comment (shubhitms) [inline example/index.js:5]: we should update the docs to reflect this
https://app.lekko.com/docs/docs/sdk/node-server#connect-with-api-client

**Review by shubhitms — APPROVED** [review]

File: example/index.js (hunk 3 above)
Comment (danlekko) [reply example/index.js:5]: docs will be updated next

**Review by DavidSGK — APPROVED** [review]

**Pull request merged**